guides/user/plugins/user-guide.md

```diff
@@ -114,122 +88,145 @@ This guide uses the [file](https://raw.githubusercontent.com/spinnaker-plugin-ex
 ]
 ```
 
+For example:
+
 ```json
 [
   {
-    "id": "examplePluginsRepo",
+    "id": "spinnaker-plugin-examples",
     "url": "https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json"
-  }
+ },
+ {
+     "id": "my-company-internal-plugins",
+     "url": "https://<my-company-internal-github>/<repo-name>/plugins.json"
+ },
+ {
+	  "id": "my-plugins",
```

Comment:
Indentation is off here.

Reply:
Huh. Weird. Indentation isn't off in my IDE. I'll delete and redo the spacing - maybe a tab snuck in there.